data/pokemon/base_stats/mew.asm

```diff
@@ -1,29 +1,20 @@
-MewBaseStats::
-db DEX_MEW ; pokedex id
-db 100 ; base hp
-db 100 ; base attack
-db 100 ; base defense
-db 100 ; base speed
-db 100 ; base special
-db PSYCHIC ; species type 1
-db PSYCHIC ; species type 2
-db 45 ; catch rate
-db 64 ; base exp yield
-INCBIN "gfx/pokemon/front/mew.pic",0,1 ; 55, sprite dimensions
-dw MewPicFront
-dw MewPicBack
-; attacks known at lvl 0
-db POUND
-db 0
-db 0
-db 0
-db 3 ; growth rate
-; learnset
-	tmlearn 1,2,3,4,5,6,7,8
-	tmlearn 9,10,11,12,13,14,15,16
-	tmlearn 17,18,19,20,21,22,23,24
-	tmlearn 25,26,27,28,29,30,31,32
-	tmlearn 33,34,35,36,37,38,39,40
-	tmlearn 41,42,43,44,45,46,47,48
-	tmlearn 49,50,51,52,53,54,55,56
-db %11111111 ; usually spacing
+	db DEX_MEW ; pokedex id
+
+	db 100, 100, 100, 100, 100
+	;   hp  atk  def  spd  spc
+
+	db PSYCHIC, PSYCHIC ; type
+	db 45 ; catch rate
+	db 64 ; base exp
+
+	INCBIN "gfx/pokemon/front/mew.pic", 0, 1 ; sprite dimensions
+	dw MewPicFront, MewPicBack
+
+	db POUND, NO_MOVE, NO_MOVE, NO_MOVE ; level 1 learnset
+	db 3 ; growth rate
+
+	; tm/hm learnset
+	tmhm 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56
```

Comment:
This much horizontal scrolling is a little excessive.

Reply:
The named constants have made that worse then. :P It could break up the single lines with backslashes:

```
	tmhm MEGA_PUNCH, MEGA_KICK, TOXIC, HORN_DRILL, BODY_SLAM, TAKE_DOWN, \
		DOUBLE_EDGE, BUBBLEBEAM, WATER_GUN, ICE_BEAM, BLIZZARD, \
		HYPER_BEAM, PAY_DAY, SUBMISSION, COUNTER, SEISMIC_TOSS, RAGE, \
		THUNDERBOLT, THUNDER, EARTHQUAKE, FISSURE, MIMIC, DOUBLE_TEAM, \
		REFLECT, BIDE, FIRE_BLAST, SKULL_BASH, REST, ROCK_SLIDE, \
		SUBSTITUTE, SURF, STRENGTH
```

Reply:
Or Mew:

```
	tmhm MEGA_PUNCH, RAZOR_WIND, SWORDS_DANCE, WHIRLWIND, MEGA_KICK, TOXIC, \
		HORN_DRILL, BODY_SLAM, TAKE_DOWN, DOUBLE_EDGE, BUBBLEBEAM, \
		WATER_GUN, ICE_BEAM, BLIZZARD, HYPER_BEAM, PAY_DAY, SUBMISSION, \
		COUNTER, SEISMIC_TOSS, RAGE, MEGA_DRAIN, SOLARBEAM, DRAGON_RAGE, \
		THUNDERBOLT, THUNDER, EARTHQUAKE, FISSURE, DIG, PSYCHIC_M, \
		TELEPORT, MIMIC, DOUBLE_TEAM, REFLECT, BIDE, METRONOME, \
		SELFDESTRUCT, EGG_BOMB, FIRE_BLAST, SWIFT, SKULL_BASH, \
		SOFTBOILED, DREAM_EATER, SKY_ATTACK, REST, THUNDER_WAVE, \
		PSYWAVE, EXPLOSION, ROCK_SLIDE, TRI_ATTACK, SUBSTITUTE, CUT, \
		FLY, SURF, STRENGTH, FLASH, UNUSED
```

Reply:
Or with 8 TMs per line instead of 80 chars:

```
	tmhm MEGA_PUNCH, RAZOR_WIND, SWORDS_DANCE, WHIRLWIND, MEGA_KICK, TOXIC, HORN_DRILL, BODY_SLAM, \
		TAKE_DOWN, DOUBLE_EDGE, BUBBLEBEAM, WATER_GUN, ICE_BEAM, BLIZZARD, HYPER_BEAM, PAY_DAY, \
		SUBMISSION, COUNTER, SEISMIC_TOSS, RAGE, MEGA_DRAIN, SOLARBEAM, DRAGON_RAGE, THUNDERBOLT, \
		THUNDER, EARTHQUAKE, FISSURE, DIG, PSYCHIC_M, TELEPORT, MIMIC, DOUBLE_TEAM, \
		REFLECT, BIDE, METRONOME, SELFDESTRUCT, EGG_BOMB, FIRE_BLAST, SWIFT, SKULL_BASH, \
		SOFTBOILED, DREAM_EATER, SKY_ATTACK, REST, THUNDER_WAVE, PSYWAVE, EXPLOSION, ROCK_SLIDE, \
		TRI_ATTACK, SUBSTITUTE, CUT, FLY, SURF, STRENGTH, FLASH, UNUSED
```

Reply:
I'm not sure. A paragraph block isn't all that readable either.